_partners/hyperspace.md

```diff
@@ -0,0 +1,65 @@
+---
+# Partner name used on the partners collection page (/partners).
+name: Hyperspace
+
+# name_long is used on the extended partner details page.
+name_long: Hyperspace
+
+# The partner logo is used on the partners collection page (/partners).
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/hyperspace_logo.png'
+
+# URL to the partner's website.
+link: 'https://www.hyper-space.io/'
+
+# logo_large is used on the extended partner details page.
+# It is normative to upload all extended partner page images to a subdirectory of /assets/media/partners/ with the name of the partner as the directory name.
+# For example given a partner named "SampleCompany" the logo_large image would be uploaded to /assets/media/partners/samplecompany/logo_large.png
+logo: '/assets/media/partners/hyperspace_logo_large.png'
```

Comment:
It looks like you might have forgotten to include hyperspace_logo_large.png with your PR.

Reply:
Hi!
Thanks for noticing. File added.

Thanks again